dspace/config/dspace.cfg

```diff
@@ -920,6 +920,11 @@ metadata.hide.dc.description.provenance = true
 # Defaults to true; If set to 'false', submitter has option to skip upload
 #webui.submit.upload.required = true
 
+# Which field should be used for type-bind
+# Defaults to 'dc.type'; If changing this value, you must also update the related
+# dspace-angular environment configuration property submission.typeBind.field
```

Comment:
Small thing... we don't need the same configuration on *both* the UI and the backend.  Instead, the UI can read this config **from** the backend, if we add this new config to `rest.properties.exposed` in `rest.cfg`.  I.e...
```
rest.properties.exposed = submit.type-bind.field
```

With that field set, this property will be accessible via the REST API at
`[dspace.server.url]/api/config/properties/submit.type-bind.field`

This would allow you to completely **remove** the `typeBind` config from the Angular side.

Reply:
This should be done